platform/source/lib/ax/platform/cluster_config/cluster_config.py

```diff
@@ -28,6 +28,7 @@
 class AXClusterConfig(with_metaclass(Singleton, object)):
 
     def __init__(self, cluster_name_id=None, aws_profile=None, config=None):
+        Cloud().set_target_cloud("aws")
```

Comment:
objects other than installer should assume these are properly set: If this class is run in a service, we export proper env so we don't need to set. If this class is run in locally (i.e. cli), cli is going to initialize it before starting any app (i.e. installer, pauser, resumer, etc)

Reply:
Clearly left over from some initial hacks. Removed.